projects/palindrome/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Palindrome in Every Language
+layout: default
+date: 2020-10-07
+last-modified: 2020-10-07
+featured-image: 
+tags: [panlindrome]
+authors:
+  - anohene1
+---
+
+This is an article on how to check if a number is a palindrome in every language.
+
+## Description
+
+A palindrome is a number that reads the same backward and forward.
+Example: 343, 121, 909, 222
+
+## Requirements
+
+Create a file called "Even Odd" using the naming convention appropriate for your language of choice.
```

Comment:
```suggestion
Create a file called "Palindrome" using the naming convention appropriate for your language of choice.
```

Reply:
My bad, I copied that sentence from an already existing file because I wanted everything to be uniform with what is already available and forgot to change it.